### PR TITLE
Make invite command

### PR DESCRIPTION
**Pull request by milanmdev**
I have made an invite command on this project. #hacktoberfest. Many people requested, and it uses "client.user.id" for the dynamic client id.

### CODE REVIEW

**Comment by SudhanPlayz** [comment]
ok looks fine to me anyways many people requested so nice

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Can you change description on the invite.js

**Comment by milanmdev** [comment]
Alright, try now

**Review by milanmdev — COMMENTED** [review]
I updated the description

File: commands/invite.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    description: "To add/invite the bot to your server",
+    usage: "[invite]",
+    aliases: ["inv"],
+  },
+
+  run: async function (client, message, args) {
+    let invite = new MessageEmbed()
+    .setTitle(`Invite ${client.user.username}`)
+    .setDescription(`Want me in your server? Invite me today! \n\n [Invite Link](https://discord.com/oauth2/authorize?client_id=${client.user.id}&permissions=8&scope=bot)`)
+    .setURL(`https://discord.com/oauth2/authorize?client_id=${client.user.id}&permissions=8&scope=bot`)
```
Comment (SudhanPlayz) [inline commands/invite.js:15]: Why permissions 8 give its correct permissions

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]
Why permissions 8 give its correct permissions on the link

**Comment by milanmdev** [comment]
Can you give me the is of what perms to give?

**Comment by SudhanPlayz** [comment]
Connecting to channels, Speaking, add reactions,  read and view voice channels and sending embeds

**Review by milanmdev — COMMENTED** [review]
Fixed again :D

File: commands/invite.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = {
     let invite = new MessageEmbed()
     .setTitle(`Invite ${client.user.username}`)
     .setDescription(`Want me in your server? Invite me today! \n\n [Invite Link](https://discord.com/oauth2/authorize?client_id=${client.user.id}&permissions=8&scope=bot)`)
```
Comment (SudhanPlayz) [inline commands/invite.js:14]: You weren't changed on the description you just changed url

**Review by SudhanPlayz — CHANGES_REQUESTED** [review]

**Comment by milanmdev** [comment]
image

**Review by milanmdev — COMMENTED** [review]
Lmao fixed again, sorry bout that. I made a variable to fix it.

**Review by SudhanPlayz — APPROVED** [review]
All good ready to merge

**Pull request merged**